e2e/tests/unit/utils/dataProvider.js

```diff
@@ -0,0 +1,91 @@
+const load = require;
+const fs = require('fs');
+const path = require('path');
+
+const loadFile = file => {
+  return Object.assign(load(`../../../../ccd-definition/${file}.json`), []);
+};
+
+let getFileData = [];
+let processDir = [];
+
+let fieldsArray = [];
+getFileData = (filePath, withConfig) => {
+  fieldsArray = [];
+  processDir(filePath, withConfig);
+  return fieldsArray;
+};
+
+processDir = (filePath, withConfig) => {
+  const fileNames = fs.readdirSync(path.resolve(__dirname, filePath));
+  if (!Object.prototype.toString.call(fileNames) === '[object Array]') {
+    const currentObject = path.resolve(__dirname, `${filePath}/${fileNames}`);
+    const stat = fs.statSync(currentObject);
+    if (stat.isFile() && !fileNames.filter(name => ['-HNL-nonprod.json', 'CUI.json'].includes(name)).length > 0) {
```

Comment:
Seeing `-HNL-nonprod.json` here. Is that intentional? I'm worried with all the variants we have people might be tempted to add bits and bobs to this dataProvider, where we should probably define a clear convention and work on that.

(I believe there were separate talks about using two suffixes: -prod and -nonprod)

Reply:
@filippo-possenti-version1 this was WIP at the time, it's now parameterized to run with all exclusion scenarios defined in build-release-definition.sh (if the exclusions are updated in that file, they should also be updated in the data provider